Src/bmpCreater.hpp

```diff
@@ -1,11 +1,85 @@
-#pragma once
-#include <cstdint>
+#include <pshpack2.h>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <Windows.h>
+/*
```

Comment:
Same problem as above. You should NOT use windows-only APIs when the content can be easily implemented with ISO C++ contents.